package.json

```diff
@@ -66,6 +67,7 @@
     "jest-environment-jsdom": "^29.4.1",
     "jotai": "^2.0.3",
     "microbundle": "^0.15.1",
+    "minimalistic-assert": "^1.0.0",
```

Comment:
```suggestion
```

Reply:
it is used in `__tests_/mutableAtom.test.tsx`

Reply:
Oops, sorry, I thought I just searched it, but didn't find it. Must have missed something.